gulpfile.js

```diff
@@ -33,4 +33,17 @@ elixir(function(mix) {
         'socialmediashare.js'
     ], 'public/js/scripts.js', 'public/js');
 
+    /**
+     * Compile sass files to css
+     */
+    mix.sass('main.scss')
+       .sass('audioplayer.scss');
```

Comment:
So essentially, If I merge this feature, everyone is expected to work from the  sass files. If there is a changed need in the audioplayer or main file, it would be the sass files that will be edited?.

The reason I'm asking is to make sure every one on the team is aware of this @andela-jkariuki